docs/fermi_hubbard/experiment_example.ipynb

```diff
@@ -667,9 +931,22 @@
   },
   "kernelspec": {
    "display_name": "Python 3",
+   "language": "python",
    "name": "python3"
+  },
+  "language_info": {
```

Comment:
Can you run this through [nbfmt](https://github.com/tensorflow/docs/blob/master/tools/tensorflow_docs/tools/) please? This normalizes the JSON and removes extra metadata which makes it easier to review diffs later.

```
$ python3 -m pip install -U --user git+https://github.com/tensorflow/docs

$ python3 -m tensorflow_docs.tools.nbfmt --indent=1 notebook.ipynb
```

Quantum folks prefer `--indent=1`